plugins/examples/change-logo/src/settings.tsx

```diff
@@ -0,0 +1,113 @@
+import { ConfigStore } from '@kinvolk/headlamp-plugin/lib';
+import { NameValueTable } from '@kinvolk/headlamp-plugin/lib/CommonComponents';
+import Box from '@mui/material/Box';
+import TextField from '@mui/material/TextField';
+import { useEffect, useState } from 'react';
+
```

Comment:
Can you please add a comment for the reader which explains what this demonstrates?

Please also add a note to the README.